part-3/pk-sim-options.md

```diff
@@ -1,54 +1,56 @@
+-   **Show software update notification if available**: When enabled and connected to the internet, a check for new versions of PK-Sim® is done automatically.
 
-* Active skin: By changing the skin in the Skins group next to the Options icon the graphical appearance of the program can be changed.
-* Number of recent file items shown: Changes the number of recent documents displayed within the File Tab. The program needs to be restarted for the changes to take effect.
-* Preferred view layout: Choose from either tabbed or accordion view in e.g. the Compound window.
-* Restore opened view when loading project: Open tabs (e.g. particular simulations, individuals or compounds) are saved upon saving and restored when re-opening the project.
-* **Show software update notification if available**: When enabled and connected to the internet, a check for new versions of PK-Sim® is done automatically.
+### Defaults
 
-## Defaults
+-   **Species** and **Population**: Changes the default species and default population used for the creation of a new individual or population.
+-   **Default parameter layout**: Changes the default parameter layout used for parameters shown within the Anatomy & Physiology tab of an individual and the Parameters tab of a simulation.
+-   **Lipophilicity**, **Fraction unbound,** and **Solubility**: You can specify the defaults for the description used for the Experiment input box when a new compound is created.
+-   **Population analysis**: After the first simulation run, an analysis window opens automatically. This option sets the default type of this first analysis ('Time Profile', 'Box Whisker', ...).
+-   **Chart y scale**: Default scaling (lin or log) of the y axis in new time-profile charts.
 
-* Default species and Default population: Changes the default species and default population used for creation of a new individual.
-* Default parameter layout: Changes the default parameter layout used for parameters shown within the Anatomy & Physiology ta b of an individual and within the Parameters tab of a simulation.
-* Lipophilicity, Fraction unbound and Solubility: You can specify the defaults for the description used for the Experiment input box when a new compound is created.
-* Population analysis: After the first simulation run an analysis window opens automatically. The default type of this first analysis ('Time Profile', 'Box Whisker', ...) is set by this option.
+### Icon Sizes
 
-## Icon Sizes
+You can change the size of the icons displayed within the **Tree view**, the **Tabs,** and the **Context menu**
 
-You can change the size of the icons displayed within the **Tree view**, the **Tabs** and the **Context menu**
+### Template Database
 
-## Template Database Path
+You can change the path to the template database by clicking on "...", or create a new template database by clicking on <img src="../assets/icons/AddAction.svg" data-size="line"/>
```

Comment:
the option is missing
<img width="313" height="116" alt="grafik" src="https://github.com/user-attachments/assets/cd565d86-d0e1-4a00-af53-26f1931a33c4" />